template/_dynamic_files.jinja2

```diff
@@ -1077,7 +1079,38 @@ When we use the base pipelines, we
 
 The first implementation of this pipeline is PATO, see https://github.com/pato-ontology/pato/blob/master/src/ontology/pato-odk.yaml.
 
-As we will test and roll out this feature more widely, we will also expand the documentation.
+To check if your ontology uses this method, check src/ontology/{{ project.id }}-odk.yaml to see if `use_base_merging: TRUE` is declared under `import_group`
+
+If your ontology uses Base Module approach, please use the following steps: 
+
+Start by declaring the term to be imported in Protege:
+
+1. Open your ontology (edit file) in Protege (5.5+).
+1. Select 'owl:Thing'
+1. Add a new class as usual.
+1. Paste the _full iri_ in the 'Name:' field, for example, http://purl.obolibrary.org/obo/CHEBI_50906.
+1. Click 'OK'
+
+<img src="https://raw.githubusercontent.com/INCATools/ontology-development-kit/master/docs/img/AddingClasses.png" alt="Adding Classes" />
+
+Next, you navigate in your terminal to the ontology directory (underneath src in your hpo root directory). 
+```
+cd src/ontology
+```
+
+Then refresh imports by running
+
+```
+sh run.sh make imports/merged_import.owl
+```
+Note: if your mirrors are updated, you can run `sh run.sh make no-mirror-refresh-merged`
+
+This requires quite a bit of memory on your local machine, so if you encounter an error, it might be a lack of memory on your computer. A solution would be to create a ticket in an issue tracker requesting for the term to be imported, and your one of the local devs should pick this up and run the import for you.
+
+Lastly, restart Protege, and the term should be imported in ready to be used.
+
+Warning: Adding declarations on editors file might cause issues if the declared class is not longer used. The declarations should be removed if no longer in use
```

Comment:
"not" --> "no"

Reply:
thanks for spell checking >.< obviously brains not in it today heh